src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -282,6 +285,8 @@ public class InstancesController : ControllerBase
                 return ExceptionResponse(exception, $"Instantiation of data elements failed for instance {instance.Id} for party {instanceTemplate.InstanceOwner?.PartyId}");
             }
 
+            await _eventsService.AddEvent("app.instance.created", instance);
```

Comment:
Where are the events we support/are planning to support documented?

Reply:
We will never be able to compile a complete list of events, but for the events produced by the AppTemplate we should update the relevant documentation. Be it about the Events component of the App itself I'm not sure.